bom/runtime/src/main/resources/extensions-support-overrides.json

```diff
@@ -0,0 +1,454 @@
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-oidc",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-openshift",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "development" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-qpid-jms",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "tech-preview" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-quartz",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-reactive-pg-client",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-rest-client",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-resteasy",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-resteasy-jackson",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-resteasy-jaxb",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-resteasy-jsonb",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-resteasy-jaxb",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-resteasy-jsonb",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-scheduler",
+      "metadata" : { 
+       "supported-by" : "redhat",
+       "redhat-support" : [
+         "supported" ]
+       }
+    },
+    {
+      "group-id": "io.quarkus",
+      "artifact-id":"quarkus-smallrye-context-propagation",
+      "metadata" : { 
+       "supported-by" : "redhat",
```

Comment:
I think sth like:
```
"support" : [
   {
      "redhat": [ "supported" ]
   },
]
```
or even:
```
"support" : [
   {
      "redhat": { "supported":true, "deprecated": true }
   }, ...
]
```
would be a bit prettier (to not have two `redhat`s in the metadata)

Reply:
@michalszynkiewicz I like your first proposal; second one about making boolean isn't really adding value besides requiring more characters ;)

After talking with @ia3andy we simplified it and suggest we go with:

```
"support" : 
   {
      "redhat": [ "supported" ]
   },
```

basically a map rather than a list.

Reply:
Not sure how the list of support elements can nicely be overridden in an additive way with our current overriding mechanism, so we would override everything.

Reply:
yeah so back and forth a few times we are ending up with 

```
"redhat-support" : [ "supported", ... ]
```

no supported-by and tools/code.quarkus.io will have a optoin to state which key to use for status.